packages/rest/src/__tests__/unit/router/openapi-path.unit.ts

```diff
@@ -42,6 +41,16 @@ describe('validateApiPath', () => {
     expect(path).to.eql('/{_foo}/{bar}');
   });
 
+  it('allows /{foo-bar}', () => {
+    const path = validateApiPath('/{foo-bar}');
+    expect(path).to.eql('/{foo-bar}');
+  });
+
+  it('allows /{foo}-{bar}', () => {
+    const path = validateApiPath('/{foo}-{bar}');
+    expect(path).to.eql('/{foo}-{bar}');
```

Comment:
Is this a valid OpenAPI path? Will our two router implementations correctly handle such path?

Reply:
Good catch. It turns out that `-` is not allowed by `path-to-regexp` as it does not match `\w`. I'll investigate more.